app/components/MinerSetupGuide.tsx

```diff
@@ -4,73 +4,100 @@ import Link from 'next/link';
+            <div>
+              <p className="font-medium">With Xverse wallet</p>
+              <p className="text-sm text-foreground/70">
+                Install{" "}
+                <Link
+                  href="https://www.xverse.app/"
+                  className="underline hover:text-foreground"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  Xverse
+                </Link>
+                {" "}→ Connect at{" "}
+                <Link
+                  href="https://parasite.space"
+                  className="underline hover:text-foreground"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  parasite.space
+                </Link>
+                {" "}→ Continue below
+              </p>
+            </div>
+          </div>
+          <div className="flex gap-3">
+            <span className="font-mono text-foreground/50 shrink-0">B.</span>
+            <div>
+              <p className="font-medium">Direct mining</p>
+              <p className="text-sm text-foreground/70">
+                Have your BTC address ready → Continue below
+              </p>
+            </div>
+          </div>
         </div>
-        <p className="text-sm mt-2">
-          <strong>Important:</strong> Use the same ending/domain (@sati.pro or @parasite.sati.pro) that appears in your generated static ln address
-        </p>
-        <p className="text-sm mt-2 italic break-words">
-          <strong>Examples:</strong><br/>
-          bc1qnotarealaddress.steveMiner.d1a7a1bef2@sati.pro<br/>
-          bc1qnotarealaddress.jillAxe.a5f9b2c8e1@parasite.sati.pro
-        </p>
-      </li>
-      <li>Use any password (it&apos;s not checked)</li>
-      <li>Save your configuration and restart if needed</li>
-    </ol>
+      </div>
+
+      <ol className="list-decimal pl-4 space-y-2 ml-6">
+        <li>Connect your Bitcoin miner and go to the &quot;Pool Settings&quot; tab</li>
+        <li>
+          Configure your mining settings:
+          <ul className="list-disc pl-4 mt-2">
+            <li>
+              Stratum URL:{" "}
+              <span className="font-mono bg-foreground/10 px-2 py-1 rounded">
+                parasite.wtf
+              </span>
+            </li>
+            <li>
+              Port:{" "}
+              <span className="font-mono bg-foreground/10 px-2 py-1 rounded">
+                42069
+              </span>
+            </li>
+            <li>
+              Depending on your miner it will look something like this:<br/>
+              <span className="font-mono bg-foreground/10 px-2 py-0.33 rounded">
+                parasite.wtf:42069<br />
+              </span>
+              <span className="font-mono bg-foreground/10 px-2 py-0.33 rounded">
+                stratum+tcp://parasite.wtf:42069
+              </span>
```

Comment:
I believe that Tailwind CSS's default spacing scales only by `0.125`/`0.25 rem`. Correct me if I am wrong, but I believe that if we use the class `py-0.33`, it will be silently ignored, resulting in no vertical padding being applied, this will not silent error because it follows Tailwind CSS's proper scales.
```suggestion
               <span className="font-mono bg-foreground/10 px-2 py-0.5 rounded inline-block">
                 parasite.wtf:42069
               </span>
               <span className="font-mono bg-foreground/10 px-2 py-0.5 rounded inline-block">
                 stratum+tcp://parasite.wtf:42069
               </span>
```